chatcommands.py

```diff
@@ -1070,7 +1070,20 @@ def report(msg, urls):
                            "one go.".format(wait))
 
     output = []
-    urls = list(set(urls.split()))
+
+    argsraw = args.split(' "')
+    urls = argsraw[0].split(' ')
+
+    # Handle determining whether a custom report reason was provided.
+    try:
+        custom_reason = ' "'.join(argsraw[1:])
```

Comment:
Seems like it might break if someone tries to provide multiple reasons.

Reply:
ArtOfCode: The only other solution is that if there are multiple items beyond 2 total items in the first split set to return an error and a failure case of "Error: that is not proper syntax for an error reason, we can only accept a single report reason".  After I am done with the weekend maintenance today at work and I am sufficiently rested, I'll see if we can "fix" this, but if I may ask in what cases do you think this might fail?  Do you have an example case?

Reply:
I had to write it this way to include the "tripleee might want to include quotes within the reason" edge case brought ul in chat.  If we are hyper restrictive then we can check the length of the first split and if it sees more than two items (URLs and reason) then throw an error back to Smokey to reply in chat that they broke the syntax.  If we want to make it as close to the original logic I wrote as possible, then we have to disallow any of the 'edge cases', which is what my initial plan was.